admission_control/admission-control-service/src/upstream_proxy.rs

```diff
@@ -12,7 +12,7 @@ use anyhow::{format_err, Result};
 use bounded_executor::BoundedExecutor;
```

Comment:
after your change, what's a use of this file?

Reply:
All the code in `upstream_proxy` that handles client txn submission will go in the next PR, which involves refactoring all the tests for this functionality